src/client/entity-editor/relationship-editor/types.ts

```diff
@@ -54,17 +72,23 @@ export type RelationshipWithLabel = {
 };
 
 export type RelationshipForDisplay = {
+	attributes: Array<Attribute>,
 	relationshipType: RelationshipType,
 	sourceEntity: Entity,
 	targetEntity: Entity,
-	rowID: number
+	rowID: string
```

Comment:
This was necessary for sorting the relationships array in redux, if I understand correctly?

Reply:
Yes exactly !!